Bottom: 73da618464d06661ec8b0936a9a17ad7237e34cb
Top:    581a9f45647d396744a0f2bc98195faa23ee2c18
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-02-16 14:22:44 +1300

Refresh of define-stream-result

---

diff --git a/testtools/testresult/real.py b/testtools/testresult/real.py
index 5a53c20..c05a002 100644
--- a/testtools/testresult/real.py
+++ b/testtools/testresult/real.py
@@ -262,7 +262,7 @@ class StreamResult(object):
     generally make a sequence of calls like::
 
       >>> result.status(self.id(), 'inprogress')
-      >>> result.status(self.id(), 'success')
+      >>> result.status(self.id(), 'finish')
 
     General concepts
     ----------------
